src/components/utils/label/label.component.tsx

```diff
@@ -0,0 +1,20 @@
+import clsx from 'clsx';
+import React from 'react';
+
+interface ILabelProps {
+  componentId: string;
+  label: string;
+}
+
+/**
+ * Label component.
+ */
+export const Label: React.FC<ILabelProps> = ({componentId, label}) => {
+  const className = clsx();
+
+  return (
+    <label className={className} htmlFor={componentId}>
```

Comment:
note that if you plan to re-use this label for radio/checkboxes inputs the `htmlFor` will/should point to nothing

Reply:
Correct, not implementing radios yet though.